bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCheckIssueVisitor.java

```diff
@@ -207,4 +207,9 @@ public void visit(ExecutableIssue issue) {
     public void visit(BlacklistIssue issue) {
         log.fine("ignored: blacklisted commit");
     }
+
+    @Override
+    public void visit(BinaryIssue issue) {
+        log.fine("ignored: binary file");
```

Comment:
Should we do something more here? Perhaps something for a follow up..

Reply:
My plan was to leave that to you :wink: